plugins/plugin-chart-table/src/DataTable/DataTable.tsx

```diff
@@ -42,8 +42,7 @@ import SelectPageSize, { SelectPageSizeProps, SizeOption } from './components/Se
 import SimplePagination from './components/Pagination';
 import useSticky from './hooks/useSticky';
 import { updateExternalFormData } from './utils/externalAPIs';
-import ServerPagination from './components/ServerPagination';
-import { ServerPage } from '../types';
+import { PAGE_SIZE_OPTIONS } from '../controlPanel';
```

Comment:
When I created the `DataTable` component, the goal was that it should be re-usable outside of the table chart, which means it should not have dependencies from outside of the `DataTable` folder. All the pagination manipulation should happen in the `TableChart` component or `transformProps`. 

`DataTable` should be a generic table renderer with pagination, search, and page size options, and that's it. It can provide hooks to control which controls to display and how to render them, but it should not contain Superset chart specific logics.

Reply:
done